src/App.tsx

```diff
@@ -20,12 +27,14 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={(event) => setQuery(event.target.value)}
```

Comment:
Move it to a function.
```suggestion
                onChange={handleQueryInput}
```